demo/app/components/selection-list/selection-list.component.html

```diff
@@ -23,14 +23,9 @@ <h3 tsCardTitle tsVerticalSpacing>
   <form
     novalidate
     fxLayout="column"
-    fxLayout.gt-sm="row"
     fxLayoutGap="1rem"
   >
 
-
-    <div fxLayout="row" fxLayoutAlign="center">
-
-      <div style="height: 42px; background: lightblue; " fxFlex="noshrink"></div>
```

Comment:
How did I not see this being added at table pr 🤦‍♀

Reply:
Same!!!